README.md

```diff
@@ -44,4 +44,12 @@ versioning provided by the golang project.  That means that git tags are created
 The following tag command will mark version `1.0.0`:
 ```
 git tag v1.0.0
-```
\ No newline at end of file
+```
+
+# Generate Cluster Role
+
+This library needs to collect the pull secret information from the openshift-config namespace.
+A template for the cluster role and role binding exist in `templates/cluster_role.yaml`.
+You must replace the `OPERATOR_SERVICE_ACCOUNT` and `OPERATOR_NAMESPACE` for the role binding.
+`make gen-cluster-role SERVICE_ACCOUNT=myoperator NAMESPACE=default` will create a `cluster_role.yaml`
```

Comment:
`make gen-cluster-role SERVICE_ACCOUNT=myoperator NAMESPACE=my-operataro-namespace` is failing for me on Fedora laptop with:
```
sed: can't read s/OPERATOR_SERVICE_ACCOUNT/myoperator/g: No such file or directory
make: *** [Makefile:16: gen-cluster-role] Error 2
```
Changing the Makefile to:
```
@sed -i 's/OPERATOR_SERVICE_ACCOUNT/$(shell printf "$(shell echo $(SERVICE_ACCOUNT))")/g' manifests/cluster_role.yaml
@sed -i 's/OPERATOR_NAMESPACE/$(shell printf "$(shell echo $(NAMESPACE))")/g' manifests/cluster_role.yaml
```
helped

Reply:
Thanks, had worked on my Mac.

Reply:
Made an update that should work for both. Feel free to verify on your Fedora laptop.

Reply:
Works good. Thx